docs/kb/semgrep-code/run-specific-version.md

```diff
@@ -0,0 +1,56 @@
+---
+tags:
+  - Semgrep CLI
+  - Semgrep in CI
+description: How to test scans with different versions of Semgrep.
+---
+
+# How to run different versions of Semgrep
+
+Most of the time, it's best to run the latest version of Semgrep, as it has the most recent features and fixes.
+
+However, when testing or managing upgrades, it can be helpful to run different versions of Semgrep to compare behavior. 
+
+:::info
+If you use Semgrep with Semgrep Cloud Platform, [only the latest 10 minor versions are supported](/docs/semgrep-cloud-platform/getting-started/).
+:::
+
+Installation with Homebrew does not support multiple versions of Semgrep, but you can use either Pip or Docker to install different versions. In the following examples, <code><span className="placeholder">x.y.z</span></code> is a placeholder for a version string.
+
+## Running different versions using pip
+
+With Pip, you can install a specific Semgrep version using pip's version syntax.
+
+<pre class="language-bash"><code>python3 -m pip install semgrep==<span className="placeholder">x.y.z</span></code></pre>
+
+Confirm installation by the following command:
```

Comment:
```suggestion
Confirm installation by entering the following command:
```

Reply:
@s-santillan Heh, I copied this exact phrase from https://semgrep.dev/docs/getting-started/. Maybe we should change both to "running" or "executing"? I looked at the style guides to see if there's a preferred word but didn't see one.

Reply:
Yup, any of those would work! I just thought it lacked a verb.

Reply:
Sounds good, I'll change to running!